src/components/Inputs/CarouselInput/CarouselInput.tsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from 'react';
+import globalStyle from '@/GlobalClasses.module.scss';
+import CircleButton from '@/components/Buttons/CircleButton/CircleButton';
+import SwitchButton from '@/components/Buttons/SwitchButton/SwitchButton';
+import Carousel from '@/components/Buttons/Carousel/Carousel';
+import { InputComponentProps } from '@/services/types';
+import { initialValues } from '@/services/initialValues';
+import { useFormContext } from 'react-hook-form';
+
+const CarouselInput = ({ title, data }: InputComponentProps) => {
+  const { setValue, getValues } = useFormContext();
```

Comment:
**nit:** `useFormContext` should not be hardcoded in such component. Just add new props as `onIncrement`, and `onDecrement`